Description: Do not impose PBR version
 The current release is too restrictive regarding PBR version, which may
 make OpenStack not installable in Sid. This patch fixes that.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2015-07-28

--- migrate-0.9.7.orig/requirements.txt
+++ migrate-0.9.7/requirements.txt
@@ -1,7 +1,7 @@
 # The order of packages is significant, because pip processes them in the order
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
-pbr>=0.11,<2.0
+pbr
 
 # never put a cap on this, *ever*, sqla versions are handled via
 # tox, and if SQLA is capped it will only make it so we aren't testing
